pmm-app/src/pmm-dbaas/components/DBCluster/EditDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -33,36 +32,40 @@ export const DBClusterAdvancedOptions: FC<FormRenderProps> = ({
   const { required, min } = validators;
   const { change } = form;
   const nodesValidators = [required, min(MIN_NODES)];
-  const resourcesValidators = [required, min(MIN_RESOURCES)];
+  const resourcesValidators = [
+    required,
+    min(0.1),
+    (value) => ((value * 10) % 1 ? 'Value must be increments of 0.1.' : undefined),
```

Comment:
I think it has to be a separate function even though we do not intend to put it into our library.

Reply:
🙏 thanks, @tiagomotasantos noticed this error too, it was removed

Reply:
No, I can't compete with @tiagomotasantos :)